hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/DistributionSetManagement.java

```diff
@@ -246,21 +249,21 @@
     /**
      * Deletes or mark as delete in case the type is in use.
      *
-     * @param type
+     * @param typeId
      *            to delete
      */
     @PreAuthorize(SpringEvalExpressions.HAS_AUTH_DELETE_REPOSITORY)
-    void deleteDistributionSetType(@NotNull DistributionSetType type);
+    void deleteDistributionSetType(@NotNull Long typeId);
```

Comment:
Now this method and some other throws a `EntityNotFoudException`. I think it is helpfull to adapt the javadoc